src/FakeItEasy/Core/EventCallHandler.cs

```diff
@@ -35,6 +41,12 @@ public void HandleEventCall(EventCall eventCall)
             }
         }
 
+        public EventCallHandler GetSnapshot()
+        {
+            var registeredHandlers = this.registeredEventHandlers.ToDictionary(kvp => kvp.Key, kvp => kvp.Value);

```

Comment:
I may be silly, but is this a fancy way to write `var registeredHandlers = new Dictionary(this.registeredEventHandlers);`?

(By "fancy", I mean "harder to read and likely less efficient".)